ui/scripts/esbuild.ts

```diff
@@ -1,16 +1,26 @@
-import { build } from "esbuild";
+/// <reference types="node" />
+import { BuildOptions, build, context } from "esbuild";
 
-build({
-  entryPoints: ["src/index.tsx"],
-  bundle: true,
-  outfile: "dist/index.js",
-  sourcemap: "inline",
-  watch: process.argv.includes("--watch"),
-  minify: !process.argv.includes("--watch"),
-  logLevel: "info",
-  loader: {
-    ".svg": "dataurl",
-    ".woff": "dataurl",
-    ".woff2": "dataurl",
-  },
-});
+async function run() {
+  const cfg: BuildOptions = {
+    entryPoints: ["src/index.tsx"],
+    bundle: true,
+    outdir: "dist",
+    sourcemap: "linked",
```

Comment:
If the UI in Core isn't super complicated, I wonder if we even need to generate source maps at all?

(I'd be fine either way, just curious what you think.)

Reply:
Sure. I changed it to only generate the sourcemap when using watch mode (for local development, where it's probably more useful)